lib/src/test/kotlin/io/bloco/faker/components/LoremTest.kt

```diff
@@ -0,0 +1,127 @@
+package io.bloco.faker.components
+
+import io.bloco.faker.Faker
+import junit.framework.TestCase.assertNotNull
+import junit.framework.TestCase.assertTrue
+import org.junit.Before
+import org.junit.Test
+
+class LoremTest {
+    private lateinit var faker: Faker
+
+    @Before
+    fun setUp() {
+        faker = Faker()
```

Comment:
I think you can use 
private val faker: Faker
   get() = Faker() 
 to always get a fresh one :) if you do inline val then for sure :D